tests/generator/test_common.py

```diff
@@ -58,15 +58,26 @@ def config_with_optional_addresses(self, eth_name, optional_addresses):
 
     def test_optional_addresses(self):
         eth_name = self.eth_name()
-        self.generate(self.config_with_optional_addresses(eth_name, '["dhcp4"]'))
-        self.assertEqual(self.get_optional_addresses(eth_name), set(["dhcp4"]))
+        self.generate(self.config_with_optional_addresses(eth_name, '["ipv4"]'))
+        self.assertEqual(self.get_optional_addresses(eth_name), set())
 
     def test_optional_addresses_multiple(self):
         eth_name = self.eth_name()
-        self.generate(self.config_with_optional_addresses(eth_name, '[dhcp4, ipv4-ll, ipv6-ra, dhcp6, dhcp4, static]'))
+        self.generate(self.config_with_optional_addresses(eth_name, '["ipv4", "ipv6", "ipv4"]'))
```

Comment:
"ipv4" is duplicated here for the same reason "dhcp4" was duplicated before: to have code coverage for the entire `normalize_yaml_tree()` method in generator/base.py :sweat_smile: